go.mod

```diff
@@ -9,8 +9,10 @@ require (
 	github.com/hashicorp/hcl/v2 v2.8.2 // indirect
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.5.0
 	github.com/mitchellh/mapstructure v1.4.1
-	github.com/nordcloud/go-pingdom v1.3.2-0.20210326061113-c7686d669f42
+	github.com/nordcloud/go-pingdom v1.3.2-0.20210406113751-d4f206ed9396
 	github.com/zclconf/go-cty v1.7.1 // indirect
 	golang.org/x/tools v0.0.0-20201028111035-eafbe7b904eb // indirect
 	google.golang.org/api v0.34.0 // indirect
 )
+
+replace github.com/nordcloud/go-pingdom => github.com/chszchen-nordcloud/go-pingdom v1.3.2-0.20210408062931-0956392a8df5
```

Comment:
I don't think this should be here anymore :)

Reply:
Sure, I was only able to change it earlier because its dependent PR(https://github.com/nordcloud/go-pingdom/pull/7) has just be merged, otherwise the build would fail.

Reply:
I will wait for the Pingdom account to be ready so that I can run the tests locally before I commit the merges.